site/formulas/decoding.md

```diff
@@ -7,14 +7,14 @@ follows.
 1. Check that `s_bytes` is the canonical encoding of a field element, or else abort.
 2. Decode `s_bytes`, a byte-string, into \\(s\\), a field element.
 3. Check that \\(s\\) is non-negative, or else abort.
-4. \\(u_1 \gets 1 - a s^{2} \\).
-5. \\(u_2 \gets 1 + a s^{2} \\).
+4. \\(u_1 \gets 1 + a s^{2} \\).
```

Comment:
I see that `u_1` is `1-as²` [here](https://github.com/dalek-cryptography/curve25519-dalek/blob/39145da396226516d8d1f663ad26f86bf04856db/src/ristretto.rs#L261). Is the code wrong too?

Reply:
Yeah, the code is updated in a different PR, it swapped naming of u_1 and u_2